transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -371,6 +368,10 @@ private void processReady(EpollEventArray events, int ready) {
                 // consume wakeup event
                 Native.eventFdRead(eventFd.intValue());
             } else {
+                if (fd == timerFd.intValue()) {
+                    continue;
```

Comment:
Based upon https://linux.die.net/man/2/timerfd_settime I wouldn't expect data to pile up. IIUC `read` just allows you to get a count of how many times the timer fired, or will block/EAGAIN, since the last time the timer was changed or since the last `read`. Since we reset the interval each time

> read(2)
If the timer has already expired one or more times since its settings were last modified using timerfd_settime(), or since the last successful read(2), then the buffer given to read(2) returns an unsigned 8-byte integer (uint64_t) containing the number of expirations that have occurred. (The returned value is in host byte order, i.e., the native byte order for integers on the host machine.)
If no timer expirations have occurred at the time of the read(2), then the call either blocks until the next timer expiration, or fails with the error EAGAIN if the file descriptor has been made nonblocking (via the use of the fcntl(2) F_SETFL operation to set the O_NONBLOCK flag).